src/hpc/thread.c

```diff
@@ -103,10 +103,93 @@ void UnlockThreadControl(void)
 #define MAP_ANONYMOUS MAP_ANON
 #endif
 
-#ifndef HAVE_NATIVE_TLS
+#ifndef USE_NATIVE_TLS
+
+#ifdef USE_PTHREAD_TLS
+static int init_tls_key = 0;
```

Comment:
Global vars (even static ones) should start with an upper case letter, and use CamelCase. Also, no need to explicitly init to 0, the standard mandates that for globals anyway.
```suggestion
static int InitTLSKey;
```

Reply:
Fixed the case. However, I've left the initialization, since it documents that this variable has a default value.